Assignment_0/200759_rahul.md

```diff
@@ -0,0 +1,55 @@
+# **Assignment 0**
+**Topic**
+1. [Commands](#Commands)
+2. [Vim](#Vim)
+3. [Git](#Git)
+4. [Github](#github)
+>## *Commands*
```

Comment:
Hey, check out the preview of a readme on VSCode by `Ctrl+Shift+V`. You'll observe that this `>` symbol messes up the file. Change it here.